python/paddle/fluid/contrib/slim/tests/test_mkldnn_int8_quantization_strategy.py

```diff
@@ -93,42 +153,59 @@ def _test_int8_quant(self, data_path, config_path):
             train_fetch_list=[],
             eval_program=None,
-            eval_reader=test_reader,
+            eval_reader=reader,
             eval_feed_list=[],
             eval_fetch_list=[],
             teacher_programs=[],
             checkpoint_path='',
             train_optimizer=None,
             distiller_optimizer=None)
         com_pass.config(config_path)
-        eval_graph = com_pass.run()
+        com_pass.run()
 
     def test_compression(self):
         if not fluid.core.is_compiled_with_mkldnn():
             return
 
-        base_url = 'http://paddle-inference-dist.bj.bcebos.com/int8/'
-        self.download_data([base_url + 'imagenet_val_100_tail.tar.gz'],
-                           ['b6e05365252f12f75e7daf3fcbc62e96'],
-                           'imagenet_val_100_tail', False)
-        data_path = os.path.join(self.cache_folder,
-                                 'imagenet_val_100_tail/data.bin')
-
-        model_urls = [
-            base_url + 'mobilenetv1_int8_model.tar.gz',
-            base_url + 'resnet50_int8_model.tar.gz'
-        ]
-        md5s = [
-            '13892b0716d26443a8cdea15b3c6438b',
-            '4a5194524823d9b76da6e738e1367881'
-        ]
-        for model_url, md5 in zip(model_urls, md5s):
-            model_name = model_url.split('/')[-1]
-            model_name = model_name.split('.')[0]
-            self.download_data([model_url], [md5], model_name)
-            config_path = self._update_config_file(model_name)
-            self._test_int8_quant(data_path, config_path)
+        int8_model_path = test_case_args.int8_model_save_path
+        data_path = test_case_args.infer_data
+        fp32_model_path = test_case_args.infer_model
+        batch_size = test_case_args.batch_size
+
+        warmup_batch_size = test_case_args.warmup_batch_size
+        quantized_accuracy = test_case_args.quantized_accuracy
+
+        _logger.info(
+            'FP32 & INT8 prediction run: batch_size {0}, warmup batch size {1}.'.
+            format(batch_size, warmup_batch_size))
+
+        #warmup dataset, only use the first batch data
+        warmup_reader = paddle.batch(
+            self._reader_creator(data_path, False),
+            batch_size=warmup_batch_size)
+        config_path = self._update_config_file(fp32_model_path, int8_model_path)
+        self._warmup(warmup_reader, config_path)
+
+        _logger.info('--- FP32 prediction start ---')
+        val_reader = paddle.batch(
+            self._reader_creator(data_path, False), batch_size=batch_size)
+        fp32_model_result = self._predict(val_reader, fp32_model_path)
+        _logger.info('--- INT8 prediction start ---')
+        val_reader = paddle.batch(
+            self._reader_creator(data_path, False), batch_size=batch_size)
+        int8_model_result = self._predict(val_reader, int8_model_path)
+        _logger.info('--- comparing outputs ---')
+        _logger.info('Avg top1 INT8 accuracy: {0:.4}'.format(int8_model_result[

```

Comment:
Please, make it printing accuracy with at least 4 digits precision after period, padding with zeros if necessary.